### PR TITLE
move eventshub to single dir

### PR DESCRIPTION
**Pull request by n3wscott**

### CODE REVIEW

File: pkg/eventshub/assert/event_info_matchers.go [hunk 1]
```diff
@@ -23,22 +23,21 @@ import (
 	cloudevents "github.com/cloudevents/sdk-go/v2"
 	cetest "github.com/cloudevents/sdk-go/v2/test"
 
-	pkgeventshub "knative.dev/reconciler-test/pkg/eventshub"
-	"knative.dev/reconciler-test/pkg/test_images/eventshub"
+	"knative.dev/reconciler-test/pkg/eventshub"
 )
 
 // EventInfo is a re-export of eventshub.EventInfo for convenience
 type EventInfo = eventshub.EventInfo
```
Comment (vaikas) [inline pkg/eventshub/assert/event_info_matchers.go:30]: Just curious if this is still necessary after the moves?

File: pkg/k8s/wait.go [hunk 2]
```diff
@@ -201,3 +200,59 @@ func WaitForAddress(ctx context.Context, gvr schema.GroupVersionResource, name s
 	})
 	return addr, err
 }
+
+// WaitForServiceHasAtLeastOneEndpoint polls the status of the specified Service
```
Comment (vaikas) [inline pkg/k8s/wait.go:204]: This comment needs to be updated.

Also, curious if we should change it to not look for exact match but for >= endpoints?

File: pkg/k8s/wait.go [hunk 3]
```diff
@@ -201,3 +200,59 @@ func WaitForAddress(ctx context.Context, gvr schema.GroupVersionResource, name s
...
+	endpointsService := client.CoreV1().Endpoints(svcNamespace)
+
+	interval, timeout := PollTimings(ctx, nil)
+	return wait.PollImmediate(interval, timeout, func() (bool, error) {
+		endpoint, err := endpointsService.Get(ctx, svcName, metav1.GetOptions{})
+		if err != nil {
+			return false, err
+		}
+
+		return countEndpointsNum(endpoint) == numOfEndpoints, nil
```
Comment (vaikas) [inline pkg/k8s/wait.go:216]: so here >= numOfEndpoints?

File: pkg/eventshub/assert/event_info_matchers.go (hunk 1 above)
Comment (n3wscott) [reply pkg/eventshub/assert/event_info_matchers.go:30]: sounds good.

File: pkg/k8s/wait.go (hunk 2 above)
Comment (n3wscott) [reply pkg/k8s/wait.go:204]: updated!

**Review by vaikas — APPROVED** [review]
/lgtm

**Pull request merged**